addons/account/models/company.py

```diff
@@ -252,6 +253,21 @@ class ResCompany(models.Model):
     )
     company_vat_placeholder = fields.Char(compute='_compute_company_vat_placeholder')
 
+    income_account_id = fields.Many2one(
+        comodel_name='account.account',
+        string="Income Account",
+        domain=ACCOUNT_DOMAIN,
+        help="This account will be used when validating a customer invoice.",
+    )
+    expense_account_id = fields.Many2one(
+        comodel_name='account.account',
+        string="Expense Account",
+        domain=ACCOUNT_DOMAIN,
+        help="The expense is accounted for when a vendor bill is validated, except in anglo-saxon"
+             " accounting with perpetual inventory valuation in which case the expense (Cost of"
+             " Goods Sold account) is recognized at the customer invoice validation.",
+    )
```

Comment:
I also don't exactly know what this helper is conveying but on product category it's same I copied from there and it was there since ages so we can check with PO and update it in both places
Probably in 18.0 in another PR :thinking: